src/frontend/api/oppsummeringvedtak.ts

```diff
@@ -7,13 +8,24 @@ export interface IVedtaksBrev {
 }
 
 export const hentVedtaksbrev = (
-    fagsakId: number,
+    fagsak: IFagsak,
```

Comment:
This works, but would it be better to let the consumer send an id. I would suggest a parameter called `behandlingId` so that it is up to the consumer.

Reply:
We need to find the active "behandling" of the "fagsak" before displaying the "vedtak". Now the logic is hidden in the component so it accepts a fagsak.
If we move out the logic, I think we have to add another consumer to extract the active "behandling" for sending to the component. Do you think it is necessary?

Reply:
I think it would be better to have an "open api" and let the consumer call it with a specific id. Another approach would be to let the backend handle this by having an endpoint called aktivVedtak/html or something like that and just pass the fagsak ID. I would probably prefer something like that. We are now leveraging the fact that we have all vedtaks in the frontend, but we might want to minimize the data flowing up to our clients.

Do you understand?

Reply:
I considered the second approach before implementation, and I thought the backend-api should be more general, so if it takes fagsak id, it cannot handle requirements like view historical behandlings of a fagsak. Thats why I made it like this.
I think I understand your both solutions, but I did not see strong reason of changing the implementation so I suggest maybe we decide and refactor it later if necessary?

Reply:
Ok, but then I think we should at least rename the function so that it clearly states that we are only getting the active vedtak.

Reply:
Agree and changed!